### PR TITLE
Add Account Recovery Events to Audit Log

### LINKED ISSUE #375: Add new recovery related events to audit log
The following new events need to be added to our audit log:

1) recovery token create

2) recovery code generated
3) recovery code used
4) recovery code failure (when user does not submit correct code)

### PR DESCRIPTION
**Pull request by rudream**
### Purpose

This PR resolves #375 

Add account recovery-related events to audit log

### Demo

image

### CODE REVIEW

File: packages/teleport/src/Audit/EventList/EventTypeCell.tsx [hunk 1]
```diff
@@ -83,6 +83,10 @@ const EventIconMap: Record<EventCode, React.FC> = {
   [eventCodes.PORTFORWARD_FAILURE]: Icons.Info,
   [eventCodes.SUBSYSTEM]: Icons.Info,
   [eventCodes.SUBSYSTEM_FAILURE]: Icons.Info,
+  [eventCodes.RECOVERY_TOKEN_CREATED]: Icons.VpnKey,
```
Comment (kimlisa) [inline packages/teleport/src/Audit/EventList/EventTypeCell.tsx:86]: this should probably be `Icons.Info` b/c it is just a subset of user token, and reset password token also uses `Icons.Info`

**Review by kimlisa — APPROVED** [review]
minor comment

File: packages/teleport/src/Audit/EventList/EventTypeCell.tsx (hunk 1 above)
Comment (rudream) [reply packages/teleport/src/Audit/EventList/EventTypeCell.tsx:86]: revised

**Comment by rudream** [comment]
### Updated Demo

image

**Pull request merged**